source/includes/masterportal/_applications.md

```diff
@@ -0,0 +1,92 @@
+### Applications
+
+Application's login can be automated by MasterPortal. It has an entrypoint that points to the MasterPortal proxy and can have credentials added to it. 
+
+#### Add Credentials
+
+```shell
+Example Request: 
+curl --request POST \
+  --url 'https://cloudmc_endpoint/rest/services/mp-devel/test-applications/apps/b6c4ca5b-e85a-41d3-80b2-5d21d6ce7060?operation=addCredentials' \
+  --header 'mc-api-key: your_api_key' \
+  --data '{
+	"username": "sample_username",
+	"password": "sample_password"
+}'
+```
+```json
+Example Response: 
+{
+  "taskId": "ac3144ab-03c8-4564-afbf-fabf5fc68199",
+  "taskStatus": "SUCCESS"
+}
+```
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/apps/:id</code>
+
+Add credentials to your own user for the given application.
+
+Required | &nbsp;
+---------- | -----
+`username`<br/>*string* | The username for the application
+`password`<br/>*string* | The password for the application
+
+#### Add Credentials for Another User
+
+```shell
+Example Request: 
+curl --request POST \
+  --url 'https://cloudmc_endpoint/services/mp-devel/test-applications/appUser/1a578977-5744-4832-bae3-d91ad2939adf:b6c4ca5b-e85a-41d3-80b2-5d21d6ce7060?operation=addUserCredentials' \
+  --header 'content-type: application/json' \
+  --header 'mc-api-key: your_api_key' \
+  --data '{
+	"cloudMcUserId": "1a578977-5744-4832-bae3-d91ad2939adf",
+	"app": {
+		"id": "b6c4ca5b-e85a-41d3-80b2-5d21d6ce7060",
+		"username": "sample_username",
+		"password": "sample_password"
+	}
+}'
+
+```
+```json
+Example Response: 
+{
+  "taskId": "ecdd71ea-1648-44d2-a9cf-71b160acab17",
+  "taskStatus": "SUCCESS"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/appUser/:id</code>
+
+Add credentials to another user for the given applications. Note the appUser's id
+is their [CloudMC user id](#administration-users) and [application](#masterportal-applications) id concatenated like such `cloudmc_user_id:application_user`.
```

Comment:
- Since the link is to `#administration-users`, maybe we can take "id" out of the link - `[CloudMC user](#administration-users)`
- concatenated like: ~such~ `cloudmc_user_id:application_user`.
- Also, since it's `cloudMcUserId ` that is used in the example and the table below, may be we can use the same key instead of `cloudmc_user_id`